test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectManagement/ObjectManagementServiceTests.cs

```diff
@@ -0,0 +1,171 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Management.Smo;
+using Microsoft.SqlTools.BatchParser.Utility;
+using Microsoft.SqlTools.Hosting.Protocol;
+using Microsoft.SqlTools.ServiceLayer.Connection;
+using Microsoft.SqlTools.ServiceLayer.IntegrationTests.Utility;
+using Microsoft.SqlTools.ServiceLayer.ObjectManagement;
+using Microsoft.SqlTools.ServiceLayer.ObjectManagement.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Scripting;
+using Microsoft.SqlTools.ServiceLayer.Scripting.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Test.Common;
+using Moq;
+using NUnit.Framework;
+using static Microsoft.SqlTools.ServiceLayer.IntegrationTests.Utility.LiveConnectionHelper;
+
+namespace Microsoft.SqlTools.ServiceLayer.IntegrationTests.ObjectManagement
+{
+    public class ObjectManagementServiceTests
+    {
+        private readonly string table_query = @"CREATE TABLE testTable1_RenamingTable (c1 int)
+                            GO
+                            ";
+        private readonly string ownerUri = "testDB";
+        private readonly string startTableName = "testTable1_RenamingTable";
+        private readonly string startColumnName = "c1";
+        private ObjectManagementService objectManagementService;
+        private SqlTestDb testDb;
+        private Mock<RequestContext<bool>> requestContextMock;
+
+        [SetUp]
+        public async Task TestInitialize()
+        {
+            this.testDb = await SqlTestDb.CreateNewAsync(TestServerType.OnPrem, false, null, table_query, "RenameTest");
```

Comment:
Instead of passing null for the values you don't care about I'd suggest using named parameters for each of the other ones (which also benefits the reader to know what the values mean without having to dig into the method signature)

Reply:
Thanks, I will remove it and I will also remove the parameters which would have the default value specified on the method.